service/installer/helm/metrics.go

```diff
@@ -55,3 +67,9 @@ func updateHelmMetrics(name string, startTime time.Time) {
 		name,
 	).Inc()
 }
+
+func reportHelmRelease(name, status string) {
+	helmReleaseFailure.WithLabelValues(
+		name, status,
```

Comment:
Only helm release name and status as prom labels. WDYT?

Reply:
If this is all we need for the alert then best to keep it simple. 👍